index.html

```diff
@@ -652,6 +653,23 @@ <h2>Pre-formatted text</h2>
 					<p><a href="#top">[Top]</a></p>
 				</footer>
 			</article>
+			<article id="text__sections">
+				<header>
+					<h2>Text Sections</h2>
+				</header>
+				<main>
+					<p>Leading Content</p>
+					<section>
+						<p>Test section 1</p>
```

Comment:
Each section needs a heading as its first element. Since these sections are a child of a container with a level 2 heading, let's use level 3 headings here.